terraform/deployments/govuk-publishing-platform/backends_origin.tf

```diff
@@ -175,6 +183,15 @@ resource "aws_lb_target_group" "signon" {
   health_check {
     path = "/healthcheck/ready"
   }
+  
+  tags = merge(
+    local.additional_tags,
+    {
+      name      = "backend-origin-signon-tg"
```

Comment:
based on my new #231, it should be just `signon-tg`

Reply:
i will have a look